packages/odata-v2/src/request-builder/update-request-builder.spec.ts

```diff
@@ -140,6 +143,86 @@ describe('UpdateRequestBuilder', () => {
     expect(actual).toEqual(entity.setOrInitializeRemoteState());
   });
 
+  it('executes the update when nullable string is set to null', async () => {
+    // Given: Entity with only key properties and one nullable string property which has non-null value
+    const keyPropGuid = uuid();
```

Comment:
[nth] consider a local helper function to build the entity and do the mock perhaps.

Reply:
I did think about this but I'm not sure. We do already have the create test entity helper, adding just a second version of that probably would not be helpful as we would need a new function for every variation of the test entity. Maybe there would be a good way to make that configurable, but this sounds like too complex code for simple test object creation imo.

Reply:
Chose to extract it into a method